### ISSUE TITLE
Using Image Component provided by Next JS

### ISSUE THREAD

**Issue by Yash-Parwani** [Clueless-Community/first-contribution#136]
Instead of using <img> html tag in contributors.js and discordhome.js, shouldnt we use <Image /> component that next js provides readily in next/image
And by using the <Image /> component helps in optimization thereby bringing down the loading time
For eg , currently while using <img> tag of HTML , the loading time of the images in contributors.js is as follows
![image](https://user-images.githubusercontent.com/95875897/193448022-8fcb7222-f9fb-4691-a3c7-d219f2d73435.png)
Using <Image /> component will help in the long run

**Comment by ramdhanihendrie** [comment]
please assign it to me

**Comment by ASANIYAN** [comment]
Hello, I would love to work on this issue

**Comment by Rajdip019** [comment]
This issue is now solved I guess. Can any one confirm?

**Comment by codeswithroh** [comment]
It's still there

**Comment by Yash-Parwani** [comment]
@Rajdip019  its still there

**Comment by Rajdip019** [comment]
Can any one mark the `img` tags where you want to use the `Image` tag in the code and send here some screenshots? 

Will be assigning this issue to him.

**Comment by codeswithroh** [comment]
- In whattodo (dir), index.js (file)
![1](https://user-images.githubusercontent.com/77570394/193684842-8b3ca2a8-2973-4d13-b81e-4cbed3bdecc6.png)

- In DiscordHome.js file
![2](https://user-images.githubusercontent.com/77570394/193685108-484729b2-2b6f-4250-97ff-1a1d66eff5b2.png)

**Comment by Rajdip019** [comment]
Go ahead @codeswithroh . Fix them and make sure. This time it's all covered and responsive as well.

**Comment by Rajdip019** [comment]
This issue is solved I guess @codeswithroh ..

**Comment by Rajdip019** [comment]
Hey, @codeswithroh is this issue completely solved? The. close this issue.

**Comment by codeswithroh** [comment]
Yeah, this is solved. In DiscordHome.js the <img /> tag is still there. But since svg is being used so, the performance won't matter.
